scripts/convex.ganache.ts

```diff
@@ -56,15 +58,15 @@ async function main(): Promise<void> {
   console.timeEnd('simulate-shutdown');
   console.log('');
 
-  // @ts-ignore looks like ganche has a bad typing
+  // @ts-ignore looks like ganache has a bad typing
   await ganache.disconnect();
 }
 
 interface PrepareGanacheOptions {
   url: string;
   blockNumber: 'latest' | number;
   blockGasLimit: string;
-  defaultBalance: string;
+  defaultBalance: number;
```

Comment:
Unrelated to this PR and mainly just a ganache question, but it feels error-prone to pass around balances as numbers since they'll will often be larger than `Number.MAX_SAFE_INTEGER`, so just curious about this change?

Reply:
It does seem silly that we expect a number when most all other options are hex strings - I'll ask around if there's a historical reason Ganache does this that I'm not aware of.